src/txkube/testing/strategies.py

```diff
@@ -20,6 +20,11 @@
     ObjectCollection,
 )
 
+# Without some attempt to cap the size of collection strategies (lists,
+# dictionaries), the slowness health check fails intermittently.  Here are
+# some sizes for collections with no other opinion on the matter.
+_AVERAGE_SIZE = 3
```

Comment:
These names seem generic enough that they sound like they're trying to globally change hypothesis' defaults? If so you might want to just do that instead via hypothesis.settings?

If not, is there a more specific name for them, or if the idea is that these are more appropriate just for this module and its strategies that they live in, maybe a comment that makes that a bit more explicit ("hey if you're writing a new strategy and you see it fail, these constants are lower than the defaults") or the like might be helpful.